backend/src/database/repositories/activityRepository.ts

```diff
@@ -218,6 +222,15 @@ class ActivityRepository {
       {
         model: options.database.member,
         as: 'member',
+        include: [
+          {
+            model: options.database.segment,
+            as: 'segments',
+            through: {
+              attributes: [],
+            },
+          },
+        ],
```

Comment:
Activities have their own segments, right? If you need segments in the returned data, why not include segments just by themselves on the top level?

Reply:
we don't need the include at all anymore, I was first keeping the affiliation settings in memberSegments then changed to a new table, this was some artifact from it. Thanks for spotting it :+1:  removed the include